### PR TITLE
Allow specification of bridges in Config.conductor & new call format

### PR DESCRIPTION
**Pull request by maackle**
- [x] my changes to the code affect some exposed aspect of the developer experience, and I have added an item to relevant 'Added, Fixed, Changed, Removed, Deprecated, or Security' heading under the 'Unreleased' heading of the CHANGELOG, with the format `- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)`

### CODE REVIEW

File: nodejs_conductor/test/test-config.js [hunk 1]
```diff
@@ -66,6 +63,84 @@ test('config construction', t => {
...
+        bridges: [
+            C.bridge('bridge-forward', instance1, instance2),
+            C.bridge('bridge-backward', instance2, instance1),
+        ],
+        debugLog: true,
+    })
+    commonConfigAssertions(t, config, path)
+    t.ok(isFullDebugLogger(config.logger))
+    t.deepEqual(config.bridges, [
+        {handle: 'bridge-forward', caller_id: 'alessia', callee_id: 'bartolini'},
```
Comment (juliolrmonteiro) [inline nodejs_conductor/test/test-config.js]: Just to make sure: the `caller_id` and `callee_id` here are instance_ids, not agent_ids, right?

Comment (maackle) [reply nodejs_conductor/test/test-config.js]: @juliolrmonteiro yes, this is the same as in the conductor config. It just so happens the instance IDs here are the same as the agent names :)

File: doc/holochain_101/src/testing_configuration.md [hunk 2]
```diff
@@ -112,36 +112,104 @@ console.log(dnaConfig)
...
+
+Takes three arguments: the bridge handle, the caller, and the callee (both instances)
+
+___
+
+**Name** handle
+
+**Type** `string`
+
+**Description** The desired bridge handle, as required by your DNA. See the [bridging section of the docs](./bridging.md) for more detail
```
Comment (Connoropolous) [inline doc/holochain_101/src/testing_configuration.md]: Can you state a very simple explanation of what the 'bridge handle' is used for?

File: doc/holochain_101/src/testing_configuration.md [hunk 3]
```diff
@@ -112,36 +112,104 @@ console.log(dnaConfig)
...
+
+___
+
+**Name** callerInstanceConfig
+
+**Type** `object`
+
+**Description** A config object with a `name` property, as produced by `Config.instance`
```
Comment (Connoropolous) [inline doc/holochain_101/src/testing_configuration.md]: Can you state a very simple explanation of what the 'caller' is, in a bridging scenario?

File: doc/holochain_101/src/testing_configuration.md [hunk 4]
```diff
@@ -112,36 +112,104 @@ console.log(dnaConfig)
...
+
+___
+
+**Name** calleeInstanceConfig
+
+**Type** `object`
+
+**Description** A config object with a `name` property, as produced by `Config.instance`
```
Comment (Connoropolous) [inline doc/holochain_101/src/testing_configuration.md]: Can you state a very simple explanation of what the 'callee' is, in a bridging scenario?

File: doc/holochain_101/src/testing_configuration.md [hunk 5]
```diff
@@ -112,36 +112,104 @@ console.log(dnaConfig)
...
+*/
+```
+
 ## Full Conductor Configuration
 
+### `Config.conductor(conductorOptions)` => `object`
+
 ### `Config.conductor(instancesArray, [conductorOptions])` => `object`
 
+There are two ways to construct a valid Conductor configuration from these `Config` helpers. Using the first way, you put all the config data into a single required object. Using the second "shorthand" style, you specify an array of `Config.instance` data, along with an optional object of extra options. The second way can be more convenient when you are just trying to set up a collection of instances with nothing extra options.
```
Comment (Connoropolous) [inline doc/holochain_101/src/testing_configuration.md]:
```suggestion
There are two ways to construct a valid Conductor configuration from these `Config` helpers. Using the first way, you put all the config data into a single required object. Using the second "shorthand" style, you specify an array of `Config.instance` data, along with an optional object of extra options. The second way can be more convenient when you are just trying to set up a collection of instances with no extra options.
```

File: doc/holochain_101/src/testing_configuration.md (hunk 4 above)
Comment (zippy) [inline doc/holochain_101/src/testing_configuration.md]:
```suggestion
**Description** A config object with a `name` property, as produced by `Config.instance`, where the name specifies which instance will be receiving calls over the bridge.
```

File: doc/holochain_101/src/testing_configuration.md (hunk 3 above)
Comment (zippy) [inline doc/holochain_101/src/testing_configuration.md]:
```suggestion
**Description** A config object with a `name` property, as produced by `Config.instance`, where the name specifies which instance will be making calls over the bridge.
```

**Comment by maackle** [comment]
@AshantiMutinta you're right, it probably should be a Result. There are two possible panics here, one for the keybundle that got added recently, and the other one is the `make_dna_config` function which actually currently cannot panic. However since we're hopefully retiring the nodejs conductor soon, maybe we can just leave it as is for now since this code will go away.

**Comment by maackle** [comment]
@Connoropolous I updated the docs using most of @zippy's suggestions

**Pull request merged**